src/main/java/com/minecolonies/coremod/entity/pathfinding/WalkToProxy.java

```diff
@@ -237,42 +239,43 @@ else if(targetY <= levelDepth)
 
     /**
      * Returns a proxy point to the goal.
-     * @param target the target.
+     *
+     * @param target         the target.
      * @param distanceToPath the total distance.
      * @return a proxy or, if not applicable null.
      */
     @NotNull
     private BlockPos getProxy(@NotNull BlockPos target, @NotNull BlockPos position, double distanceToPath)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "getProxy" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)